staker/staker.go

```diff
@@ -387,6 +436,13 @@ func (s *Staker) Start(ctxIn context.Context) {
 		}
 		return s.config.StakerInterval
 	})
+	s.CallIteratively(func(ctx context.Context) time.Duration {
+		err := s.checkLatestConfirmed(ctx)
```

Comment:
I prefer having it in the same thread as checkLatestStaked. Not critical, but would behave a little better when L1 is rate limiting/unresponsive.